### PR TITLE
Move benefits education to benefits claims section with same URI

### PR DESCRIPTION
**Pull request by mchristiansonVA**
## Summary

- *This work is behind a feature toggle (flipper): NO*
- Moves the benefits education betamocks configuration section to the benefits claims config section which uses the same URI.

## Related issue(s)

[API-40354](https://jira.devops.va.gov/browse/API-40354)

## Testing done

- [ ] *New code is covered by unit tests*
- The services config section added for Benefits Education has the same base_uri (https://sandbox-api.va.gov/) as the Benefits Claims section below, but the configuration doesn't consolidate or cascade based on the base_uri. For benefits claims it's matching the base_uri for Benefits Education, but not matching any of the Benefits Claims endpoints defined in the section below, thus breaking all mocking for Benefits Claims in Sandbox.
- Tested calls to POA v1 2122 with lighthouse_claims_api_poa_use_bd flipper enabled and the following settings defined for BD:
benefits_documents:
    use_mocks: true
    host: https://sandbox-api.va.gov
- Confirmed we're getting the expected mocked response back from BD for these calls:
{"data"=>{"success"=>true, "requestId"=>407}}

## What areas of the site does it impact?
Benefits Claims mocking configuration
Benefits Education mocking configuration

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by jerekshoe — DISMISSED** [review]

File: config/betamocks/services_config.yml [hunk 1]
```diff
@@ -866,6 +855,13 @@
       :cache_multiple_responses:
         :uid_location: "url"
         :uid_locator: "\/services/veteran_verification/v2/disability_rating\/(.+)"
+    # Benefits Education
+    - :method: :get
+      :path: '/services/benefits-education/v1/education/chapter33'
+      :file_path: "/lighthouse/benefits_education"
+      :cache_multiple_responses:
+      :uid_location: "query"
+      :uid_locator: "icn"
```
Comment (FonzMP) [inline config/betamocks/services_config.yml:864]: need indented I believe.

**Review by AdamKing0126 — DISMISSED** [review]
In theory this looks ok to me, although the benefits education stuff is not really related to benefits claims.  Maybe the section can be just named "Lighthouse Benefits", so as not to be confusing to future devs?

File: config/betamocks/services_config.yml [hunk 2]
```diff
@@ -819,17 +819,6 @@
...
-  :endpoints:
-  - :method: :get
-    :path: '/services/benefits-education/v1/education/chapter33'
-    :file_path: "/lighthouse/benefits_education"
-    :cache_multiple_responses:
-      :uid_location: "query"
-      :uid_locator: "icn"
-
 # Lighthouse Benefits Claims API
 - :name: "Lighthouse Benefits Claims"
```
Comment (jerekshoe) [inline config/betamocks/services_config.yml:823]: I would rename this to Lighthouse and add a comment above the first entry that says `# Benefits Claims`

**Comment by FonzMP** [comment]
> In theory this looks ok to me, although the benefits education stuff is not really related to benefits claims. Maybe the section can be just named "Lighthouse Benefits", so as not to be confusing to future devs?

agreed @AdamKing0126 - the original implementation was done by @jerekshoe when he was building out the connection to our API (Benefits Claims) and it's since been expanded. Once we (Team Dash on Lighthouse) realized we didn't get the "cascade" of configuration values, we added our BD section to the existing (and I think it's been expanded on since as well), but I do agree that we could just blanket it to some generic Lighthouse name instead.

**Review by FonzMP — APPROVED** [review]
:shipit:

**Review by ryan-mcneil — APPROVED** [review]

**Pull request merged**